.github/workflows/deploy.yml

```
@@ -176,4 +176,63 @@
+      
+      - name: Download version-map
+        uses: actions/download-artifact@v4
+        with:
+          name: contracts
+          path: "artifacts"
+
+      - name: Extract version map
+        run: |
+          cd artifacts
+          cat version_map.json
+          cp version_map.json ../version_map.json  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)